pyresample/test/test_bucket.py

```diff
@@ -139,7 +144,8 @@ def test_get_count(self):
 
     def test_get_average(self):
         """Test averaging bucket resampling."""
-        data = da.from_array(np.array([[2., 4.], [2., 2.]]))
+        data = da.from_array(np.array([[2., 4.], [3., np.nan]]),
```

Comment:
I already added a NaN to the data here, so that should cover the count issue.

Reply:
Oh, right! Missed that when browsing the changed rows.